packages/overlay/src/vaadin-overlay-position-mixin.js

```diff
@@ -93,12 +93,24 @@ export const PositionMixin = (superClass) =>
           type: Boolean,
           value: false,
         },
+
+        /**
+         * If the overlay content has no intrinsic height, this property can be used to set
+         * the minimum vertical space required by the overlay. If there is not enough space
+         * available on the primary side, the overlay will be positioned on the opposite side.
+         *
+         * @attr {number} required-vertical-space
+         **/
+        requiredVerticalSpace: {
```

Comment:
Not 100% sure about the property name. Any other suggestions? It's private so it doesn't matter all that much.

Reply:
Sounds fine. I would suggest to mention in the JsDoc that this effectively disables the content measurement in favor of using this fixed value for determining the open direction.

Reply:
Updated the description